src/threadpool.c

```diff
@@ -160,20 +160,13 @@ static void post(struct uv__queue* q, enum uv__work_kind kind) {
 }
 
 
-#ifdef __MVS__
-/* TODO(itodorov) - zos: revisit when Woz compiler is available. */
-__attribute__((destructor))
-#endif
```

Comment:
Removing this attribute will prevent uv__threadpool_cleanup() from running during shutdown on z/OS, due to the change [here](https://github.com/libuv/libuv/blob/v1.x/src/uv-common.c#L977-L982). We need to determine why that change was required. Please note there's currently no external CI/CD runs for libuv, which explains why you haven't encountered a regression with this PR.

Reply:
Isn't zOS running uv__os390_cleanup() instead?

Reply:
z/OS runs uv__os390_cleanup() as part of the cleanup, and uv__threadpool_cleanup() needs to be called after main() ends.